CMakeLists.txt

```diff
@@ -578,6 +578,7 @@ if(NOT UNIX)
       if(HAVE_SCHANNEL_H)
         set(USE_SCHANNEL ON)
         set(SSL_ENABLED ON)
+	check_library_exists_concat("crypt32" CertFreeCertificateContext HAVE_LIBCRYPT32)
```

Comment:
Please fix the indentation by adding one additional space in front of this line.